### PR TITLE
2022 02 26 dynamic fee gui

### PR DESCRIPTION
**Pull request by Christewart**
Fix bug where our frontend desktop GUI had a hard coded fee rate of 50 sats/vbtyte. This PR calls our backend to determine the appropriate fee rate for network conditions.

### CODE REVIEW

**Comment by benthecarman** [comment]
It wasn't hard coded it was a `var` that was set on start up

File: app/gui/src/main/scala/org/bitcoins/gui/GlobalData.scala [hunk 1]
```diff
@@ -133,7 +135,16 @@ object GlobalData {
...
-  var feeRate: FeeUnit = SatoshisPerVirtualByte.fromLong(50)
+  def getFeeRate: FeeUnit = {
+    val feeRateT = ConsoleCli
+      .exec(EstimateFee, GlobalData.consoleCliConfig)
+      .map { feeStr =>
+        val feeUnit = SatoshisPerVirtualByte(Satoshis(feeStr.toLong))
+        feeUnit
+      }
+
+    feeRateT.get
```
Comment (benthecarman) [inline app/gui/src/main/scala/org/bitcoins/gui/GlobalData.scala:146]: maybe should have a cache incase this fails

**Review by benthecarman — COMMENTED** [review]
This looks better than the previous impl

**Comment by Christewart** [comment]
> This looks better than the previous impl

I think we broke the previous impl in #3973 also. I believe thats why my DLCs are always using 50 sats/vbyte in fees

Somehow the exception was being swallowed

**Comment by benthecarman** [comment]
>  I believe thats why my DLCs are always using 50 sats/vbyte in fees

rekt

**Pull request merged**